conda_pack/tests/test_core.py

```diff
@@ -407,29 +459,36 @@ def test_dest_prefix(tmpdir, py36_env):
 
     # shebangs are rewritten using env
     with tarfile.open(out_path) as fil:
-        text_from_conda = fil.extractfile('bin/conda-pack-test-lib1').read()
-        text_from_pip = fil.extractfile('bin/pytest').read()
-        binary_from_conda = fil.extractfile('bin/clear').read()
+        lib1_script = '/'.join([BIN_DIR, 'conda-pack-test-lib1'])
+        pytest_binary = 'scripts/pytest.exe' if on_win else 'bin/pytest'
```

Comment:
Would be good to use `BIN_DIR` here as well.

Reply:
We have an uppercase/lowercase issue here. `BIN_DIR` alone won't actually work; we'll have to convert it to lowercase somehow.

Reply:
This isn't the only place we have to deal with that though so I'll clean that up

Reply:
I've implemented a solution. Yes, it's because of case normalization, but that's far too convenient to forego most of the time.